src/Learning/KWDataUtils/PLMTDatabaseTextFile.h

```diff
@@ -220,11 +220,12 @@ class PLMTDatabaseTextFile : public KWMTDatabaseTextFile
 
 	// Memorisation des index des attributs pour les mappings pour la serialisation
 	// Un vecteur d'index est memorise pour chaque mapping
-	// ainsi qu'un vecteur des index des attributs de cle dans le cas de classes racines
+	// ainsi qu'un vecteur des index des attributs de cle dans le cas de classes princicipaleq,
```

Comment:
s/principaleq/principales/

Reply:
Corrigé

Reply:
We still have "princi**ci**pales"; the second "ci" should be removed.

Reply:
Corrigé (désolé, je n'avais pris en compte qu'une seule des deux typos du même mot)